core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py

```diff
@@ -92,7 +92,7 @@ def save(self, file_path: pathlib.Path) -> None:
 
         with open(file_path, "w", encoding="utf-8") as settings_file:
             logger.debug(f"Saving settings on: {file_path}")
-            settings_file.write(Pykson().to_json(self))
+            settings_file.write(Pykson().to_json(self, base_indent=4))
```

Comment:
uhg, maybe our version doesn't have base_indent? Also, what is the difference between `base_indent` and `indent`?

Reply:
Thanks! But we have a fork of pykson that is full of changes that are not merged and the project appears to be dead at the moment. We are moving to pydantic, and we should move all projects, removing pykson in the end. This just adds a minor quality for us when checking logs. 

https://github.com/sinarezaei/pykson/pull/19